index.js

```diff
@@ -310,8 +325,9 @@ module.exports = class Hyperdrive extends ReadyResource {
     await Promise.allSettled(proms)
   }
 
-  entries (opts) {
-    return this.files.createReadStream(opts)
+  entries (opts = {}) {
```

Comment:
lets make this (range, opts) like readstream in bee now

Reply:
Went with

```
  entries (range = {}, opts = {}) {
    if (opts) opts.keyEncoding = FILES_SUB
    else range.keyEncoding = FILES_SUB // backward compat

    return this.db.createReadStream(range, opts)
  }

```
to keep backward compat